engine/gfx/sprites.asm

```diff
@@ -98,7 +98,7 @@ DoNextFrameForFirst16Sprites:
 	ld l, a
 	ld h, HIGH(wVirtualOAMSprite16)
 
-.loop2 ; Clear (wVirtualOAM + [wCurSpriteOAMAddr] --> Sprites + $40)
+.loop2 ; Clear (wVirtualOAM + [wCurSpriteOAMAddr] --> Sprites + OAMF_YFLIP)
```

Comment:
Is this supposed to be replaced? It seems more like an offset, but then I don't get why `Sprites` and not `wVirtualOAM`. (Which also should better be called `wShadowOAM`, as per widespread usage.)

Reply:
Replacing the name of that WRAM constant is better left for a separate PR first and then merged back into here, that will be done as soon as I get back today

Reply:
Agreed about `wVirtualOAM`, but the `$40` is still a problem.

Reply:
Comment fixed